src/xdocs/writingjavadocchecks.xml.vm

```diff
@@ -23,25 +23,63 @@
 
     <section name="What is Javadoc comment">
       <p>
-      Javadoc comment is multiline comment <code>/*  */</code> that starts with <b>*</b> character and placed above class definition, interface definition, enum definition, method definition or field definition.
+      Javadoc comment is a multiline comment <code>/*  */</code> that starts with the <b>*</b> character and placed above the class definition, interface definition, enum definition, method definition or field definition.
+      If an annotation precedes any of the definitions listed above, then the javadoc comment should be placed before the annotation.
+      If several multiline comments with javadoc identifiers are placed sequentially, only the nearest on top (above) multiline comment with the javadoc identifier will be used.
+      </p>
+      <p>
+      Javadoc comment contains of: short summary (first sentence), documentation section, tag section. The first sentence has a special meaning and should be clear and punchy, short and ended by a dot symbol.
```

Comment:
`Javadoc comment contains of:` => `Javadoc comments contain:`
`clear and punchy, short and ended by` => `clear, punchy, short, and is ended by`
`dot symbol` => `period symbol`

> Javadoc comments contain

Should this be `should contain` as these area aren't really required otherwise we wouldn't need the `JavadocMethod` check? Tag section is completely optional if there are no parameters/returns/throwns.

Reply:
done